### PR TITLE
[LINGI1131] Add new summary

### PR DESCRIPTION
**Pull request by Peiffap**
These were the temporary commit messages.

[LINGI1131] Summary: Add CM1

[LINGI1131] summary2: rework CM1

[LINGI1131] summary2: final fixes to CM1

[LINGI1131] summary2: add CM2

[LINGI1131] summary2: add cm3

[LINGI1131] summary2: add cm4

[LINGI1131] summary2 : add cm5 + fixes up to 4

[LINGI1131] summary2: add logic equations

[LINGI1131] summary2: add bounded buffer

[LINGI1131] test2019: add statements

[LINGI1131] summary2: add cm7

[LINGI1131] Add CM8

Add cm9

Add CM10 + CM11

[LINGI1131] Add CM12

### CODE REVIEW

File: src/q6/oz-INGI1131/summary2/img/567Tree.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...54320988,0,0)" xlink:href="#m" id="f"/><path fill="#3d4752" d="M110-160c48 1 74 30 74 79 0 53-28 85-80 85-65 0-83-55-86-122-5-90 50-162 133-122 14 7 22 21 27 39l-31 6c-5-40-67-38-82-6-9 19-15 44-15 74 11-20 30-34 60-33zm-7 138c34 0 49-23 49-58s-16-56-50-56c-29 0-50 16-49 49 1 36 15 65 50 65" id="n"/><use transform="matrix(0.07407407407407408,0,0,0.07407407407407408,0,0)" xlink:href="#n" id="g"/><path fill="#3d4752" d="M64 0c3-98 48-159 88-221H18v-27h164v26C143-157 98-101 97 0H64" id="o"/><use transform="matrix(0.07407407407407408,0,0,0.07407407407407408,0,0)" xlink:href="#o" id="h"/><path fill="#3d4752" d="M117-194c89-4 53 116 60 194h-32v-121c0-31-8-49-39-48C34-167 62-67 57 0H25l-1-190h30c1 10-1 24 2 32 11-22 29-35 61-36" id="p"/><path fill="#3d4752" d="M24-231v-30h32v30H24zM24 0v-190h32V0H24" id="q"/><path fill="#3d4752" d="M24 0v-261h32V0H24" id="r"/><g id="i"><use transform="matrix(0.07407407407407408,0,0,0.07407407407407408,0,0)" xlink:href="#p"/><use transform="matrix(0.07407407407407408,0,0,0.07407407407407408,14.814814814814817,0)" xlink:href="#q"/><use transform="matrix(0.07407407407407408,0,0,0.07407407407407408,20.666666666666668,0)" xlink:href="#r"/></g></defs></g></svg>
```
Comment (MartinBraquet) [inline src/q6/oz-INGI1131/summary2/img/567Tree.svg]: Is this file mandatory?

File: src/q6/oz-INGI1131/test/2019/Mars/All/Makefile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+SOL=both
```
Comment (MartinBraquet) [inline src/q6/oz-INGI1131/test/2019/Mars/All/Makefile]:
```suggestion
```

File: src/q6/oz-INGI1131/summary2/img/567Tree.svg (hunk 1 above)
Comment (Peiffap) [reply src/q6/oz-INGI1131/summary2/img/567Tree.svg]: Yes, it's an image I use in the document iirc.

**Comment by MartinBraquet** [comment]
@Jimvy , on merge ?

**Pull request merged**